NewRelicExampleApp/React-Native-Test-App/__tests__/App-test.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  renderer.create(<App />);
```

Comment:
```suggestion
  // renderer.create(<App />);
```
We're running into some test configuration errors here with unknown imports. It causes our agent's test suite to fail due to this. I think it'd be fine to comment out this test since this is just a sample app and it'll resolve this issue.

Reply:
Sounds good, just commented out that line in [this](https://github.com/newrelic/newrelic-react-native-agent/pull/106/commits/918e795c0a561ee934738ffd8fa335968cfa04ba) commit